lexicons/com/atproto/admin/defs.json

```diff
@@ -127,144 +167,221 @@
             "#recordViewNotFound"
           ]
         },
-        "reportedBy": {"type": "string", "format": "did"},
-        "createdAt": {"type": "string", "format": "datetime"},
-        "resolvedByActions": {"type": "array", "items": {"type": "ref", "ref": "com.atproto.admin.defs#actionView"}}
+        "reportedBy": { "type": "string", "format": "did" },
+        "createdAt": { "type": "string", "format": "datetime" },
+        "resolvedByActions": {
+          "type": "array",
+          "items": { "type": "ref", "ref": "com.atproto.admin.defs#actionView" }
+        }
       }
     },
     "repoView": {
       "type": "object",
-      "required": ["did", "handle", "relatedRecords", "indexedAt", "moderation"],
+      "required": [
+        "did",
+        "handle",
+        "relatedRecords",
+        "indexedAt",
+        "moderation"
+      ],
       "properties": {
-        "did": {"type": "string", "format": "did"},
-        "handle": {"type": "string", "format": "handle"},
-        "email": {"type": "string"},
-        "relatedRecords": {"type": "array", "items": {"type": "unknown"}},
-        "indexedAt": {"type": "string", "format": "datetime"},
-        "moderation": {"type": "ref", "ref": "#moderation"},
-        "invitedBy": {"type": "ref", "ref": "com.atproto.server.defs#inviteCode"},
-        "invitesDisabled": {"type": "boolean"}
+        "did": { "type": "string", "format": "did" },
+        "handle": { "type": "string", "format": "handle" },
+        "email": { "type": "string" },
+        "relatedRecords": { "type": "array", "items": { "type": "unknown" } },
+        "indexedAt": { "type": "string", "format": "datetime" },
+        "moderation": { "type": "ref", "ref": "#moderation" },
+        "invitedBy": {
+          "type": "ref",
+          "ref": "com.atproto.server.defs#inviteCode"
+        },
+        "invitesDisabled": { "type": "boolean" }
+      }
+    },
+    "subjectRepoView": {
+      "type": "object",
+      "nullable": [
```

Comment:
Since all of these properties are marked as optional, we have two ways of representing missing data (omitted or `null`), and it might be nice to reduce that down to just one possibility if we can.  It also feels as though we might want some minimal amount of information that should be able to be provided here, whereas now you could receive an object that is empty or only contain `null` values.